### DISCUSSION TITLE
Filebrowser on dietpi Raspberry 4b

### DISCUSSION THREAD

**Discussion by AJD1000** [filebrowser/filebrowser#2937] [Q&A]
Just installed Filebrowser on my Raspberry 4b using the built in software installation menu, it seemed to install without error and I can open the Filebrowser web interface and see the login page, but I can't log in with the default usename and password admin and admin, or admin and admin123, it says "wrong credentials" can anyone help me with this.

**Comment by gletiecq** [comment]
When I did this I had to add the user credentials using filebrowser command line interface because the default install didn't have any users installed by default (possibly because I was initially trying to use noauth).  When the session timed out after two hours (the default) it presented me with a login screen but no users were established, so I was locked out.  

This may have been just the wonky way I initially tried to set this up, and perhaps you stumbled on that method as well.  If all else fails, do a "filebrowser config init" to zero-out any unintended foolishness and see if that restores the default admin user.  If that's still a problem, "filebrowser users add <username> <password>" should at least establish a default user-level account you can get into, and a "filebrowser users update <username> --perm.admin" should be the correct syntax to make that user an admin.  From there, the settings menu should help you get most of the rest of the setup you need done.

Hope this helps.

**Reply by gletiecq** [reply]
Apparently the markup tried to hurt me there.  Here are the commands in full:

Zero out a filebrowser configuration: `filebrowser config init`
Add a user: `filebrowser users add myusername mypassword`
Make a user admin: `filebrowser users update myusername --perm.admin`

**Reply by AJD1000** [reply]
many thanks gletiecq, because I had insatlled FileBrowser via Dietpi's menu, the default username/password were different, I was pointed to this info which worked. https://dietpi.com/docs/software/cloud/#__tabbed_14_1

**Reply by gletiecq** [reply]
Well, that's convenient.  Definitely a plus for dietpi!

**Comment by gletiecq** [comment]
When it comes time to establish filebrowser as a systemd service that will start on boot, here's what I did:

New file: /etc/systemd/service/filebrowser.service

```
[Unit]
Description=filebrowser service
After=network.target

[Service]
Type=simple
Restart=on-failure
ExecStart=/usr/local/bin/filebrowser -d /etc/filebrowser/filebrowser.db  --token-expiration-time=12h
RestartSec=1

[Install]
WantedBy=multi-user.target
```

setup commands:
```
systemctl enable filebrowser.service
systemctl daemon-reload
mkdir /etc/filebrowser
cp filebrowser.db /etc/filebrowser
systemctl start filebrowser.service
```
